akka-javasdk/src/main/java/akka/javasdk/http/HttpResponses.java

```diff
@@ -189,5 +191,28 @@ public static HttpResponse notImplemented(String text) {
     return ok(text).withStatus(StatusCodes.NOT_IMPLEMENTED);
   }
 
+  private final static ContentType TEXT_EVENT_STREAM = ContentTypes.parse("text/event-stream");
+
+  /**
+   * @return A HttpResponse with a server sent events (SSE) stream response. The HTTP response will contain each element
+   *         in the source, rendered to JSON using jackson. An SSE keepalive element is emitted every 10 seconds if the stream
```

Comment:
```suggestion
   *         in the source, rendered to JSON using jackson. A SSE keepalive element is emitted every 10 seconds if the stream
```

Reply:
Actually is correct because of the abbreviation "ess" starts with a vowel (just like "An HTML page")